db/db.go

```diff
@@ -1818,7 +1818,12 @@ func updateQueueDeposits() error {
 	// then we add any new ones that are queued
 	_, err = WriterDb.Exec(`
 		INSERT INTO validator_queue_deposits
-		SELECT validatorindex FROM validators WHERE activationepoch=9223372036854775807 and status='pending' ON CONFLICT DO NOTHING`)
+		SELECT validatorindex, activationeligibilityepoch 
+		FROM validators 
+		WHERE activationepoch=9223372036854775807 and status='pending' 
+		ON CONFLICT (validatorindex) DO UPDATE SET
+			activationeligibilityepoch = EXCLUDED.activationeligibilityepoch
+		`)
 	if err != nil {
 		logger.Errorf("error adding queued publickeys to validator_queue_deposits: %v", err)
 		return err
```

Comment:
Don't you also need to set the **activationeligibilityepoch** when you set the **block_slot** and **block_index** 5 lines below this?